lib/wor/paginate/formatter.rb

```diff
@@ -32,8 +32,13 @@ def paginated_content
       end
 
       def serialized_content
-        return paginated_content.as_json if serializer.blank?
-        paginated_content.map { |item| serializer.new(item) }
+        return paginated_content.map { |item| serializer.new(item) } if serializer.present?
+
+        if defined? ActiveModelSerializers::SerializableResource
+          ActiveModelSerializers::SerializableResource.new(paginated_content).as_json
```

Comment:
Have you tested what happens if the gem can't find a serializer for the paginated_content? does it raise an exception?